NuRadioMC/utilities/Veff.py

```diff
@@ -610,29 +380,29 @@ def get_Veff_single(filename, trigger_names, trigger_names_dict, trigger_combina
                 if("Vrms" in values['efficiency']):
                     Vrms = values['efficiency']['Vrms']
                 e = get_efficiency(max_amplitudes / Vrms)  # we calculated the maximum amplitudes for all gids, now we select only those that triggered
-                Veff = V * np.sum(weights[triggered] * e) / n_events
+                Veff = volume_proj_area * np.sum(weights[triggered] * e) / n_events
+                Vefferror = 0
+                if(np.sum(weights[triggered]) > 0):
+                    Vefferror = Veff / np.sum(weights[triggered] * e) ** 0.5
+                FC_low, FC_high = FC_limits(np.sum(weights[triggered] * e))
+                Veff_low = volume_proj_area * FC_low / n_events
+                Veff_high = volume_proj_area * FC_high / n_events
 
-            Vefferror = 0
-            if(np.sum(weights[triggered]) > 0):
-                Vefferror = Veff / np.sum(weights[triggered]) ** 0.5
-            FC_low, FC_high = FC_limits(np.sum(weights[triggered]))
-            Veff_low = V * FC_low / n_events
-            Veff_high = V * FC_high / n_events
-            out['Veffs'][trigger_name] = [Veff, Vefferror, np.sum(weights[triggered]), Veff_low, Veff_high]
+            out[veff_aeff][trigger_name] = [Veff, Vefferror, np.sum(weights[triggered]), Veff_low, Veff_high]
     return out
 
 
 def tmp(args):
-    return get_Veff_single(*args)
+    return get_Veff_Aeff_single(*args)
```

Comment:
Why is this function necessary? Can't you just call get_Veff_Aeff_single directly?

Reply:
there is a tiny difference. tmp unpacks the arguments with the `*` operator